client/mysqlslap.c

```diff
@@ -1793,6 +1819,11 @@ run_scheduler(stats *sptr, statement *stmts, uint concur, ulonglong limit)
 
   con.stmt= stmts;
   con.limit= limit;
+  if (qps_target)
+    con.srv_time_target= 1000000 * concur / qps_target > ULONG_MAX ? ULONG_MAX :
```

Comment:
client/mysqlslap.c:1825:56: warning: comparison of constant 18446744073709551615 with expression of type 'unsigned int' is always false [-Wtautological-constant-out-of-range-compare]
    con.srv_time_target= 1000000 * concur / qps_target > ULONG_MAX ? ULONG_MAX :

This needs to get fixed.